services/web-frontend/internal/src/App.tsx

```diff
@@ -38,15 +39,22 @@ class App extends React.Component<unknown, StateType> {
     // const { client } = this.state;
     return (
       <div className="app">
-        <Topbar />
         <Router>
           <Switch>
-            <Route path="/">
+            <Route exact path="/">
+              <Topbar />
               <Storyline />
             </Route>
+            <Route path="/login">
+              <Topbar />
+              <Login />
+            </Route>
+            <Route path="/register">
+              <Topbar />
+              <Registration />
+            </Route>
           </Switch>
         </Router>
-        <Footer />
```

Comment:
Hey, it looks like you merged rather than rebased, and in the process moved the footer usage.